source/index.html.md

```diff
@@ -1,164 +1,644 @@
 ---
-title: API Reference
+title: Elemeno Documentation
 
 language_tabs:
-  - rest
-  - js
-  - php
+  - http
 
 includes:
-  - errors
 
 search: false
 ---
 
 # Introduction
 
-Welcome to the Kittn API! You can use our API to access Kittn API endpoints, which can get information on various cats, kittens, and breeds in our database.
+NEED SOME STUFF HERE
 
-We have language bindings in Shell, Ruby, and Python! You can view code examples in the dark area to the right, and you can switch the programming language of the examples with the tabs in the top right.
+# Single Items
 
-This example API documentation page was created with [Slate](https://github.com/tripit/slate). Feel free to edit it and use it as a base for your own API's documentation.
+Single Items are items that standalone and do not belong to a collection. They have their own unique content model, and the item is returned via the API as a single resource. Single items are perfect for things like an "About Us" or "Contact" pages on a website, or the landing screen of an application. Any content that will exist on it's own can be thought of as a Single Item. 
 
-# Authentication
+# Collections
 
-> To authorize, use this code:
+Collections are collections of multiple items with the same content model. Items are returned from the API as an array and each item can be accessed individually via it's ID or "slug". Collections also support searching and filtering so you can find a specific subset of items from a collection based on your query parameters.
```

Comment:
- `it's` in the second sentence should be `its`
- we don't _really_ support searching yet, just filtering. Maybe the sentence could be something like:
> Collections support filtering to find a specific subset of items from a collection based on your query parameters.